internal/cli/auth/login.go

```diff
@@ -139,7 +206,8 @@ func LoginBuilder() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.isGov, "gov", false, "Log in to Atlas for Government.")
 	cmd.Flags().BoolVar(&opts.isCloudManager, "cm", false, "Log in to Cloud Manager.")
 	cmd.Flags().BoolVar(&opts.noBrowser, "noBrowser", false, "Don't try to open a browser session.")
-
+	cmd.Flags().BoolVar(&opts.noProfile, "loginOnly", false, "Skip profile configuration.")
```

Comment:
[nit] why not using similar names in `noProfile` and `loginOnly`?

Reply:
it started as `noProfile` but then `loginOnly` felt as a better name, renamed the prop now